Makefile

```diff
@@ -16,10 +30,6 @@ submodules:
 
 FAIL_BUILD_CVSS_LIMIT ?= 0
 
-.PHONY: security-check
-security-check: security-report
-	mvn org.owasp:dependency-check-maven:check -DassemblyAnalyzerEnabled=false -DfailBuildOnCVSS=$(FAIL_BUILD_CVSS_LIMIT)
-
 .PHONY: security-report
```

Comment:
this should be removed too

Reply:
I dont this needs to be removed..

Reply:
This is calling the `mvn org.owasp:dependency-check-maven:check` plugin task, the same thing called by the new dependency check, but without the correct config or suppressions file.

Reply:
ok done..